src/index.ts

```diff
@@ -42,6 +44,17 @@ export {
   flush,
 }
 
+export namespace CliUx {
```

Comment:
For some reason naming the export `CliUx` feels weird to me.

I feel like `Ux` might be better but then you have `Ux.ux` which also feels weird to me.

Reply:
They both feel weird to me.